package sicv.lib.exceptions;

public final class ExceptionMessages {

	public static final String EXC_NULL = "Null Value";

	public static final String EXC_CLASS_NOT_FOUND = "Class Not Found";

	public static final String EXC_FAIL_GET_COMUNICATION_CHANNEL = "Fail to Try Recover a Comunication Channel";

	public static final String EXC_FAIL_RELEASE_COMUNICATION_CHANNEL = "Fail to Try Release a Comunication channel";

	public static final String EXC_CONNECT = "Failure to Connect";

	public static final String EXC_FAIL_DISCONNECT = "Failure to Disconnect";

	public static final String EXC_TRANSACTION_ACTIVE = "Already a Transaction Active";

	public static final String EXC_FAIL_TRANSACTION = "Transaction Fail";

	public static final String EXC_ALREADY_EXIST = "Object already exist";

	public static final String EXC_FAIL_BEGIN_SEARCH = "Problems to Begin Search";

	public static final String EXC_FAIL_QUERY = "Probleamas in Query";

	public static final String EXC_FAIL_NEXT_OBJET = "Unable to Sign Next";

	public static final String EXC_FAIL_UPDATE = "Problems in Upadte";

	public static final String EXC_FAIL_INCLUSION = "Problems in Inclusion";

	public static final String EXC_FAIL_EXCLUSION = "Problems in Exclusion";

	public static final String EXC_FAIL_DATA_BASE = "There was error in the database";

	public static final String EXC_FAIL_INSERTION_VOLATILE = "Error to Insert in Volatile Collection ";

	public static final String EXC_FAIL_SITUATION = "This complaint's status is closed";

	public static final String EXC_FAIL_CLOSE_TRANSACTION = "The Transaction can not be Closed";

	public static final String EXC_FAIL_UPDATE_COPY = "Error upgrade. Possibly the object being accessed concurrently, restart the process.";
}